test/e2e/e2e.yaml

```diff
@@ -0,0 +1,106 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+setup:
+  env: kind
+  file: kind.yaml
+  steps:
+    - name: install cert-manager
+      command: |
+        # kind k8s cluster is in $TMPDIR
+        cp $TMPDIR/e2e-k8s.config ~/.kube/config
+        kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
+      wait:
+        - namespace: cert-manager
+          resource: pod
+          for: condition=Ready
+    - name: install operator
+      command: |
+        export OPERATOR_IMG=controller:v0.4
+        make operator-docker-build   
+        kind load docker-image controller:v0.4
+        make operator-install
+        make operator-deploy
+      wait:
+        - namespace: skywalking-swck-system
+          resource: pod
+          for: condition=Ready
+    - name: wait webhook installing
+      command: |
+        sleep 30
+    - name: setup oapserver and ui
+      command: |
+        kubectl create -f config/operator/samples/default.yaml
+      wait:
+        - namespace: default
+          resource: OAPServer/default
+          for: condition=Available
+        - namespace: default
+          resource: UI/default
+          for: condition=Available
+    - name: setup java agent demo
+      command: |
+        kubectl label namespace default swck-injection=enabled
+        sed 's/oapserver-clusterip/'$(kubectl get svc default-oap -o go-template --template='{{.spec.clusterIP}}')'/' test/e2e/demo.yaml | kubectl create -f -
```

Comment:
Please deploy skywalking components to another namespace instead of `default`.

For example, `skywalking-system`. If that, pls to use`skywalking-system-oap.skywalking-system:11800` to set up the backend address of the injected agent.

Reply:
get, thanks.